src/mumble/VoiceRecorder.cpp

```diff
@@ -311,7 +311,7 @@ void VoiceRecorder::run() {
 			break;
 		}
 
-		while (!m_abort && !m_recordBuffer.isEmpty()) {
+		while (!(m_config.mixDownMode && m_config.transportEnable) && !m_abort && !m_recordBuffer.isEmpty()) {
```

Comment:
I think at this point this could use some named constants that give names to parts of the while condition such that the overall condition is easier to read as a whole.

Reply:
As in breaking up the condition into pieces pre-loop?

Reply:
Exactly